packages/compiler/emitter-framework/type-emitter.ts

```diff
@@ -670,7 +670,8 @@ export class TypeEmitter<T, TOptions extends object = Record<string, never>> {
         case "Operation":
         case "Enum":
         case "Union":
-          return this.emitter.emitDeclarationName(t);
+          const declName = this.emitter.emitDeclarationName(t);
+          return declName[0].toUpperCase() + declName.slice(1);
```

Comment:
Should we be providing in the compiler or a library those naming helpers?

Reply:
Do you mean `Capitalize` here? Otherwise, this is a compiler/library since it's the emitter framework!

Reply:
yeah meant captialize but think we'd probably want pascalCase, camelCase, etc.. to
ah right yeah then maybe just export from the emitter framework, I can file an issue